helpers/translate.js

```diff
@@ -226,6 +226,7 @@ module.exports = {
     creditNotesNotFound: 'Credit notes not found.',
     creditNoteNotFound: 'Credit note not found.',
     creditNoteUpdated: 'Credit note updated.',
+    creditNoteNotCompani: 'Credit note was not created by Compani.',
```

Comment:
en fait en suivant la logique des autres il faut juste mettre `Credit note not created by Compani.`

Reply:
C'est juste